### PR TITLE
George2

### PR DESCRIPTION
**Pull request by AdamRison**

### CODE REVIEW

File: src/main/java/resources/templates/scripts/scripts.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+// George Totolos 7-2-2015
+
+
+function getProductTable()
```
Comment (AdamRison) [inline src/main/java/resources/templates/scripts/scripts.js]: You need to add comments and a lot of them.  I can guess at what it is that you're doing in this file, but the point is that I shouldn't have to.

File: src/main/java/resources/templates/scripts/scripts.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+                rowCount = rowCount + 1;
+
+            }
+        }, error: function (error) {
+            alert("Could not retrieve Product data");
+        }
+    });
+}
+function getShipmentsTable()
```
Comment (AdamRison) [inline src/main/java/resources/templates/scripts/scripts.js]: This function needs comments too.  What we're doing should be clear for internal quality.

File: src/main/java/resources/templates/shipmentcalc.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>Getting Started: Serving Web Content</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+</head>
+<body>
+<h1>Shipment Calculations</h1>
+
+<p th:text="${errorMessage}" />
+<p th:text="'Total Cost:'+ ${totalCost}" />
```
Comment (AdamRison) [inline src/main/java/resources/templates/shipmentcalc.html]: Are you replacing the Java class methods that we made to do the heavy lifting?  I know html comments are weird, but you should definitely add some to make your code clear.

File: src/main/java/resources/templates/styles/style.css [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+    -webkit-border-radius: 12px;
+    -moz-border-radius: 12px;
+    border-radius: 12px;
+    -moz-box-shadow: 1px 1px 3px #000;
+    -webkit-box-shadow: 1px 1px 3px #000;
+    box-shadow: 1px 1px 3px #000;
+}
+.close:hover {
+    background: #00d9ff;
+}
```
Comment (AdamRison) [inline src/main/java/resources/templates/styles/style.css]: Pretty looking stuff.

File: src/main/java/resources/templates/weightCalculation.html [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+   <p th:text="'Price per unit: '+${pricePerUnit}" />
+   <!--<output th:value="${pricePerUnit}" th:field="*{pricePerKg}" />-->
+    Confirm Price Per Kg: <input type="text" th:field="*{pricePerKg}" />
+    <br> </br>
+    Enter shipment id: <input type="text" th:field="*{shipmentId}" />
+<br> </br>
+    <button type="submit">Update Shipment</button>
+    <button onclick="location.href='/weightCalc'">Cancel</button>
+</form>
+
```
Comment (AdamRison) [inline src/main/java/resources/templates/weightCalculation.html]: Again, are you using excluding the Java code that we made to do the calculations?  There's nothing wrong with either but we need to discuss it as a group.  We've all put in work on various aspects of this and if you move away from the group, then our work becomes worthless.  Also, if this is what you're doing, did you run any tests to make sure that this and other functions/methods work?  What about edge cases?

File: src/main/resources/templates/scripts/scripts.js [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+
+            }
+        }, error: function (error) {
+            alert("Could not retrieve Shipment data");
+        }
+    });
+}
+
+
+
```
Comment (AdamRison) [inline src/main/resources/templates/scripts/scripts.js]: Add some comments.

**Comment by AdamRison** [comment]
You need more comments and tests.  Some of your code is unclear as it stands and without tests can only be considered unsafe.  We can't give a non-tested product to the client.  We should a group discuss about how to proceed.

File: src/main/java/resources/templates/scripts/scripts.js (hunk 1 above)
Comment (get13) [inline src/main/java/resources/templates/scripts/scripts.js]: Yeah this file (and the css files) still need prettied up a bit

File: src/main/java/resources/templates/shipmentcalc.html (hunk 3 above)
Comment (get13) [inline src/main/java/resources/templates/shipmentcalc.html]: The shipmentCalc file was left over from the initial creation of the entire project, it will not be included.

File: src/main/java/resources/templates/weightCalculation.html (hunk 5 above)
Comment (get13) [inline src/main/java/resources/templates/weightCalculation.html]: Same as the shipmentCalc file, this was just left over and is not a part of my submission
